rl_loop.py

```diff
@@ -49,20 +54,36 @@ def print_flags():
                     for flag, value in flags.items()))
 
 
+def get_models():
+    all_models = gfile.Glob(os.path.join(MODELS_DIR, '*.meta'))
+    model_filenames = [os.path.basename(m) for m in all_models]
+    return [(shipname.detect_model_num(m), shipname.detect_model_name(m))
+            for m in model_filenames]
+
+
 def get_latest_model():
     """Finds the latest model, returning its model number and name
 
     Returns: (17, 000017-modelname)
     """
-    all_models = gfile.Glob(os.path.join(MODELS_DIR, '*.meta'))
-    model_filenames = [os.path.basename(m) for m in all_models]
-    model_numbers_names = [
-        (shipname.detect_model_num(m), shipname.detect_model_name(m))
-        for m in model_filenames]
+    model_numbers_names = get_models()
     latest_model = sorted(model_numbers_names, reverse=True)[0]
     return latest_model
 
 
+def enough_games_to_train():
+    model_numbers_names = get_models()
+    if len(model_numbers_names) <= 2:
```

Comment:
Not sure I understand this condition

Reply:
with just the bootstrap model, you don't have a 2nd-most-recent... so just go ahead and train one on whatever you've got and then relax.